src/codeu/chat/server/Server.java

```diff
@@ -206,7 +209,10 @@ public void run() {
           final int type = Serializers.INTEGER.read(connection.in());
           final Command command = commands.get(type);
 
-          if (command == null) {
+          if (type == NetworkCode.SERVER_INFO_REQUEST) {
+            Serializers.INTEGER.write(connection.out(), NetworkCode.SERVER_INFO_RESPONSE);
+            Uuid.SERIALIZER.write(connection.out(), info.version);
```

Comment:
Thanks for this clarification! I was super confused when I was doing this because it didn't seem right. I think they changed the tutorial after I had already put this in my code.